x-pack/plugins/security_solution/public/common/components/exceptions/exceptionable_linux_fields.json

```diff
@@ -0,0 +1,4 @@
+[
+    "file.path",
```

Comment:
The Endpoint will be processing these as `exact_cased`, correct?

Reply:
yes, that's my understanding.

@ferullo @gabriellandau does that sound right?  We'll need to add a Linux artifact which we'll disallow users from selecting `caseless`.

Reply:
@peluja1012 @ferullo @gabriellandau just confirmed that we'll generate `exact_cased` when using `file.path`

looks like this:
```json
{
  "entries": [
    {
      "type": "simple",
      "entries": [
        {
          "field": "file.path",
          "operator": "included",
          "type": "exact_cased",
          "value": "c/user"
        }
      ]
    }
  ]
}
```